docs/cloud/introduction/index.md

```diff
@@ -0,0 +1,18 @@
+---
+id: index
+title: Introduction to Temporal Cloud
+description: Learn more about Temporal Cloud's operating envelope, pricing, and support.
+sidebar_label: Introduction
```

Comment:
the subpages feel a little hidden to me. like i don't know if i would click on introduction to find this info

<img width="296" alt="image" src="https://github.com/temporalio/documentation/assets/251288/394adc88-8f18-4de5-9f89-7d34109c0b24">

Reply:
Noted. We still want to try this out.